music/src/music_rnn/model.ts

```diff
@@ -155,13 +163,18 @@ export class MusicRNN<A extends controls.ControlSignalUserArgs> {
    * @param steps How many steps to continue.
    * @param temperature (Optional) The softmax temperature to use when sampling
    * from the logits. Argmax is used if not provided.
-   * @param controlSignalArgs (Optional) Arguments for creating control tensors.
+   * @param chordProgression (Optional) Chord progression to use as
+   * conditioning.
    */
   async continueSequence(
       sequence: INoteSequence, steps: number, temperature?: number,
-      controlSignalArgs?: A): Promise<INoteSequence> {
+      chordProgression?: string[]): Promise<INoteSequence> {
     sequences.assertIsQuantizedSequence(sequence);
 
+    if (this.chordEncoder && !chordProgression) {
+      throw new Error('Chord progression expected but not provided.');
+    }
```

Comment:
Also add an error if one is provided but not expected.

Reply:
Done.